docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/2423/2423_5.md

```diff
@@ -0,0 +1,64 @@
+# Check disabled Hot node pools autoscaling in Maintenance mode
+
+Test verifies that Hot node pools autoscaling is disabled in the maintenance mode.
+
+**Prerequisites**:
+- Admin user
+- Perform [2423_1](2423_1.md) case
+
+**Preparations**:
+1. Open the **Cluster state** page
+2. Click the **HOT NODE POOLS** tab
+3. Click the "**+ Create**" button
+4. Specify follow values 
+   - *Pool name*
+   - *Starts on* day - the current day of the week then specify the *Starts on* time `00:00`
+   - *Ends on* day - the next day of the week then specify the *Ends on* time `23:59`
+   - *Region* - *default_region*
+   - `spot` type for the *Price type*
+   - *Instance type*
+   - *Disk* size - `20`
+   - Mark *Autoscaled* checkbox and set follow values for autoscalling
+     - *Min Size* - `2`
+     - *Max Size* - `4`
+     - *Scale Up Threshold* - 70
+     - *Scale Step* - 1
+5. Click the **+ Add docker image** button
+6. In the appeared field, select test tool
+7. Click the **CREATE** button 
+8. Click the **Refresh** button every minute until nodes appear in the node list and 2 running nodes appears in the pool state
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Open the **Tools** page | | 
+| 2 | Select test tool | |
+| 3 | At the tool page, hover over the **Run v** button | |
+| 4 | Click the **Custom settings** button in the list | |
+| 5 | Expand the **Exec environment** section | |
+| 6 | Set *Node type* the same as *Instance type* specified for node pool at step 4 of the Preparations | |
+| 7 | Set *Disk* as `15` | | 
+| 8 | Set *Cloud Region* the same as *Region* specified for node pool at step 4 of the Preparations | |
+| 9 | Expand the **Advanced** section | | 
+| 10 | Select **Spot** Price type | |
+| 11 | Launch the run | |
+| 12 | At the **Runs** page, click the just-launched run | |
+| 13 | Expand the **Instance** section. Wait until the **IP** label appears | | 
+| 14 | Open the **Cluster state** page | |
+| 15 | Click the **HOT NODE POOLS** tab | **1/2** nodes is shown in the pool state for pool created at step 7 of the Preparations |
+| 16 | Click the node pool created at step 7 of the Preparations | One node has the label with the run ID launched at step 11 | 
+| 17 | Repeat steps 1-15 | **2/3** nodes is shown in the pool state for pool created at step 7 of the Preparations |
```

Comment:
**2/3** nodes label doesn't appear right away. Possible, here the "waiting" step should be added?

Reply:
Done